wallet/how-to/get-started-building/run-devnet.md

```diff
@@ -5,46 +5,84 @@ sidebar_position: 2
-
-:::caution important
-Your seed phrase controls all your accounts, so we recommend keeping at least one seed phrase for
-development, separate from any used to store real value.
-You can manage multiple seed phrases by using multiple browser profiles, each with its own
-MetaMask installation.
-:::
-
-In the **Server** setting of your workspace, find the hostname and port of your Ganache
-network, which comprises the RPC URL of your network:
-
-```text
-http://<hostname>:<port>
-```
-
-In the MetaMask extension, connect to your Ganache network:
-
-1. Select the network you're currently connected to.
-1. Select **Add network**.
-1. Select **Add a network manually**.
-1. Enter the RPC URL of your network.
-1. Enter MetaMask's default [chain ID](../connect/detect-network.md#chain-ids) for Ganache, `1337`.
+## Connect to Hardhat
+
+Follow these steps to connect MetaMask to a Hardhat development network.
+
+1. Follow the [Hardhat Quick Start](https://hardhat.org/hardhat-runner/docs/getting-started#quick-start)
+    to set up a Hardhat development network.
+
+2. Create a new
+    [MetaMask seed phrase](https://support.metamask.io/hc/en-us/articles/360060826432-What-is-a-Secret-Recovery-Phrase-and-how-to-keep-your-crypto-wallet-secure#:~:text=Your%20Secret%20Recovery%20Phrase%20(SRP,are%20connected%20to%20that%20phrase.))
+    specifically for development.
+
+    :::caution important
+    Your seed phrase controls all your accounts, so we recommend keeping at least one seed phrase for
+    development, separate from any used to store real value.
+    You can manage multiple seed phrases by using multiple browser profiles, each with its own
+    MetaMask installation.
+    :::
+
+3. In your `hardhat.config.js` file, in the `networks.hardhat` configuration:
+
+    - Specify your MetaMask seed phrase in the
+      [`accounts`](https://hardhat.org/hardhat-network/docs/reference#accounts) field.
+
+      :::tip
+      Alternatively, to prevent committing your seed phrase, we recommend adding your seed phrase to a
+      [`.env` file](https://docs.infura.io/tutorials/developer-tools/javascript-dotenv) and using the
+      `process.env` global variable in `hardhat.config.js`.
+      :::
+   
+    - Specify the [chain ID `1337`](https://hardhat.org/hardhat-network/docs/metamask-issue) in the
+      [`chainId`](https://hardhat.org/hardhat-network/docs/reference#chainid) field.
+
+    ```js title="hardhat.config.js"
+    module.exports = {
+      networks: {
+        hardhat: {
+          accounts: {
+            mnemonic: "<your MetaMask seed phrase>",
+          },
+          chainId: 1337
+        },
+      },
+    };
+    ```
+
+    Hardhat automatically gives each of your first 20 accounts 10000 test ether (you can modify
+    these numbers in the [`accounts`](https://hardhat.org/hardhat-network/docs/reference#accounts)
+    configuration), which makes it easy to start development.
```

Comment:
```suggestion
        ```js title="hardhat.config.js"
        module.exports = {
          networks: {
            hardhat: {
              accounts: {
                mnemonic: "<your MetaMask seed phrase>",
              },
              chainId: 1337
            },
          },
        };
        ```

        Hardhat automatically gives each of your first 20 accounts 10000 test ether (you can modify
        these numbers in the [`accounts`](https://hardhat.org/hardhat-network/docs/reference#accounts)
        configuration), which makes it easy to start development.
    ```

Reply:
Added indentation so this aligns with the numbered bullet

Reply:
I think the existing indentation was correct here.